tasks/phylogenetic_inference/augur/task_augur_mutation_context.wdl

```diff
@@ -0,0 +1,37 @@
+version 1.0
+
+task mutation_context {
+  input {
+    File refined_tree
+    File ancestral_nt_muts_json
+    String build_name
+    
+    Int cpu = 4
+    Int memory = 64
+    Int disk_size = 100
+    String docker = "us-docker.pkg.dev/general-theiagen/theiagen/nextstrain-mpox-mutation-context:2024-06-27"
+  }
+  command <<<
+    # capture version information
+    
+
```

Comment:
might be helpful to add 2 DEBUG statements before and after calling the python script. The log file in Terra is blank except for the cromwell messages.

Could you please add `echo "DEBUG: Running mutation_context.py script now..."` to line 17

and add `echo "DEBUG: Finished running mutation_context.py script."` after calling the script?